packages/transformers/js/src/visitors/process.js

```diff
@@ -30,6 +32,8 @@ export default {
       } else {
         morph(node, types.booleanLiteral(true));
       }
+      deleteClosestFalsyBranch(ancestors);
+      asset.setAST(ast);
```

Comment:
Not 100% sure this should be added here as I am not 100% aware what it does. But it looked logically to not just have it for `process.env` but also for `process.browser`.